doc/admin-guide/logging/understanding.en.rst

```diff
@@ -157,3 +157,19 @@ aforementioned aggregate functions and the specification of an interval, as so:
 The interval itself is given with *n* as the number of seconds for each period
 of aggregation. There is no default value.
 
+Logging Performance
+-------------------
+
+In normal operations, log entries are strictly ordered in the output file.
+This serialization of entries comes at a cost as multiple threads potentially
+contend for log access.  For binary logs or when order does not matter, ATS supports
+faster logging where each thread can buffer its own entries.  In this mode, log
+parsers will need to expect out of order entries, but ATS can log much larger
+transaction rates.
+
+.. code:: yaml
+
+   formats:
+   - name: mysummary
+     format: '%<operator(field)> , %<operator(field)>'
+     fast: true
```

Comment:
Should we link to this section?  https://docs.trafficserver.apache.org/admin-guide/logging/formatting.en.html#timestamps-and-durations .  Since timestamping would be even more important if log record order corresponded less to request order.

Reply:
I'm wondering if we should mention that increasing this, https://docs.trafficserver.apache.org/admin-guide/files/records.config.en.html?highlight=records%20config#proxy-config-log-log-buffer-size, will increase the speedup seen from using `fast`.